lang/ecl/Portfile

```diff
@@ -6,7 +6,7 @@ PortGroup           compiler_blacklist_versions 1.0
 
 name                ecl
 version             23.9.9
-revision            0
+revision            1
```

Comment:
from the description it sounds like this is fixing a build issue? If so, no "revision" increase is required.

Reply:
good point